tests/safety/common.py

```diff
@@ -634,6 +638,8 @@ def test_reset_angle_measurements(self):
     # reset sample_t by reinitializing the safety mode
     self.setUp()
 
+    # rx one to update sample_t
+    self.assertTrue(self._rx(self._angle_meas_msg(0)))
```

Comment:
why is the RX necessary?

Reply:
because receiving a message calls `update_sample` which sets the min and max from the non-zero sample array

Reply:
that should also be reset on safety mode init

Reply:
That's what this does https://github.com/commaai/panda/pull/1376/files#diff-9e71d0935391e66564e1fe068f0e9c0b7a2523e5a31c96d2556205ee60b6d15aR347-R352

Reply:
done a little cleaner